src/index.html

```diff
@@ -7,8 +7,87 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="use-credentials">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <div class="class"> -->
+    <header class="header">
+      <img
+          class="logo"
+          src="./images/logo.png" alt="">
+      <nav class="navBig">
+        <ul class="list">
+          <li class="is-active">
+            <a href="">apple</a>
+          </li>
+          <li class="nav__link">
+            <a class="link" href="">samsung</a>
+          </li>
+          <li class="nav__link">
+            <a class="link" href="">смартфоны и телефоны</a>
+          </li>
+          <li class="nav__link">
+            <a
+            class="link__active"
+            href="" data-qa="hover">
+            ноутбуки и компьютеры
+            </a>
```

Comment:
```suggestion
            <a
              class="link__active"
              href="#"
              data-qa="hover"
            >
              ноутбуки и компьютеры
            </a>
```
Відступи.
Кожен атрибут має бути на новому рядку, якщо їх 3 і більше